app/helpers/ncr/work_orders_helper.rb

```diff
@@ -0,0 +1,16 @@
+module Ncr
+  module WorkOrdersHelper
+    def approver_options
+      # @todo should this list be limited by client/something else?
+      # @todo is there a better order? maybe by current_user's use?
```

Comment:
Maybe putting ones they've used before first would be good, but this is fine for now.